src/components/landing/Information.jsx

```diff
@@ -0,0 +1,25 @@
+import Image from "next/image";
+import image from "/public/transparent_logo_1.webp";
```

Comment:
don't import the image like this, when it is in the public folder you can reference it as `/transparent_logo_1.webp`
for more information: https://nextjs.org/docs/app/building-your-application/optimizing/static-assets

Reply:
When I look at older UCR projects, they imported the image like this and then formatted it and I was just wondering why not doing this is better?